OrbitQt/main.cpp

```diff
@@ -212,6 +214,11 @@ FigureOutDeploymentConfiguration() {
   }
 }
 
+static void DisplayErrorToUser(const std::string& message) {
```

Comment:
Why std::string? This leads to unnecessary conversions between UTF-16 and UTF-8. It also adds abseil as an additional dependency in this file. I don't see the advantage.

Reply:
Fair Point. I somehow thought we are preferring std::string and absl::StrFormat over QString. But I guess since we have Qt as a dependency here, we can use it anyways. Im going change it

Reply:
We definitely do. But we can't get around QString here anyway.